daisy_workflows/build-publish/rhui/rhua_sync_status.py

```diff
@@ -0,0 +1,154 @@
+#  Copyright 2022 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Publish metrics for RHUA sync status."""
+
+import os
+import subprocess
+import time
+
+import requests
+
+
+REPODIR = '/var/lib/rhui/remote_share/symlinks/pulp/content/content'
```

Comment:
i think some of this doesn't pass PEP8, can you run that to confirm?

Reply:
The only output I get from flake8 is that it doesn't like the Google-standard 2 space indentation instead of the 4 spaces. The sort-of weird import order is due to the fact that "requests" is not built-in, but is a third-party package (that's already installed on our RHUA machines).